client/src/Pages/Social.jsx

```diff
@@ -86,7 +87,45 @@ const Social = () => {
   const [currentUser, setCurrentUser] = useState(null);
   const [users, setUsers] = useState(null);
   const [matches, setMatches] = useState(null);
+  const [messages, setMessages] = useState(null);
+  const [messageText, setMessageText] = useState('');
   const [friends, setFriends] = useState(null);
+  const [recipient, setRecipient] = useState(null);
+  const [currentConversation, setCurrentConversation] = useState(null);
+  const scrollRef = useRef();
+
+  const handleMessageChange = (e) => {
+    console.log(e.target.value);
```

Comment:
lots of console logs in this PR. do you still need any of these?

Reply:
I do not.